lib/getWorkspacesYarn.js

```diff
@@ -39,5 +46,19 @@ function getWorkspacesYarn(cwd) {
 	return workspaces;
 }
 
+/**
+ * Ignore node_modules and only push ignored workspace.packages if no args have been passed via CLI.
+ * @param {string[]} packages  Manifest packages set on workspaces attribute.
+ * @param {string[]|null} ignorePackages (Optional) Packages to be ignored passed via cli.
+ * @returns {string[]} An array packages to be ignored in the semver release process.
+ */
+function _setIgnoredPackages(packages, ignorePackages = null) {
```

Comment:
We can remove this code, right?

Reply:
I hope so. Removed it, but codeclimate might complain about cognitive complexity.

Reply:
@antongolub FYI: https://codeclimate.com/github/qiwi/multi-semantic-release/pull/42